content/en/docs/languages/java/getting-started.md

```diff
@@ -146,9 +146,14 @@ number of ways, the steps below use environment variables.
    export JAVA_TOOL_OPTIONS="-javaagent:PATH/TO/opentelemetry-javaagent.jar" \
      OTEL_TRACES_EXPORTER=logging \
      OTEL_METRICS_EXPORTER=logging \
-     OTEL_LOGS_EXPORTER=logging
+     OTEL_LOGS_EXPORTER=logging \
+     OTEL_METRIC_EXPORT_INTERVAL=15000
    ```
 
+   Note that `OTEL_METRIC_EXPORT_INTERVAL=15000` (milliseconds) is a temporary
+   setting to test that your metrics are properly generated. Remember to remove
+   the setting once you are done testing. The default is 60000 milliseconds.
+
    {{% alert title="Important" color="warning" %}}Replace `PATH/TO` above, with
    your path to the JAR.{{% /alert %}}
```

Comment:
A few suggestions:

1. Move the "note" into the note that's already there, immediately below.
2. Let's keep things as DRY as we can and only name the env var, not it's values (default or otherwise).

Is this good for you @cartermp et al.?

```suggestion
   {{% alert title="Important" color="warning" %}}
   
   - Replace `PATH/TO` above, with your path to the JAR.
   - Set `OTEL_METRIC_EXPORT_INTERVAL` to a value well below the default, as we illustrate above, **only during testing** to help you more quickly ensure that metrics are properly generated.
   {{% /alert %}}
```

Reply:
Thanks! Applied and ran prettier.

Reply:
Yep, all good